ivadomed/config/config_pose.json

```diff
@@ -0,0 +1,94 @@
+{

```

Comment:
I've cleaned up this PR a little, and fixed a few issues that were preventing the testing commands from running. My hope is that this will provide a fresh base for us to work from. :slightly_smiling_face: 

I'm also happy to help out with whatever the next steps will be (e.g. training/evaluating the model using `spine-generic`).